eng/pipelines/official.yml

```diff
@@ -132,8 +132,8 @@ extends:
               - task: 1ES.DownloadPipelineArtifact@1
                 displayName: 🟣 Import PackageArtifacts from Previous Stage
                 inputs:
-                  artifactName: PackageArtifacts
-                  targetPath: $(Pipeline.Workspace)/PackageArtifacts
+                  artifactName: Artifacts_Windows_NT_Release
+                  targetPath: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release
```

Comment:
Why was the artifact name changed? And this name seems arbitrary (meaning, it has no relationship to the build being created). We don't build any binaries so naming it this is misleading.